### DISCUSSION TITLE
css files for event not found

### DISCUSSION THREAD

**Discussion by carolune** [pretalx/pretalx#1135] [Self-hosting] [answered]
Hi,

I have a (seemingly) working installation of pretalx, and I have created an event. It works, except that no css is loading. When I look at the logs, I see the following error: 

`WARNING 2021-07-08 14:29:31,974 django.request log Not Found: /media/myevent/cfp.somecode.css
WARNING 2021-07-08 14:29:31,976 django.request log Not Found: /media/myevent/agenda.someothercode.css
`

When I look for the files, they are there, and there are no problems with permissions.
I have checked that pretalx is set up to reach the right media folder (/data/media/) and it is.

The admin interface works fine, and can access whatever .js and .css files it needs.

I am at a complete loss...

System: Ubuntu 18.04, using mysql

**Comment by rixx** [comment, marked as answer]
pretalx doesn't deliver files from the media directory itself, you have to configure your web server to do it for you. You can see an example here in the nginx configuration in the docs: https://docs.pretalx.org/administrator/installation.html#step-7-ssl

**Reply by carolune** [reply]
Thank you so much! After much, much trawling on the web to try to get the apache equivalent of the nginx alias setting work, I finally found this goldmine that resolved it for me. It may be useful for others:
https://djangodeployment.readthedocs.io/en/latest/05-static-files.html

**Reply by rixx** [reply]
Thank you for sharing this! I'm sadly not fluent in Apache configs at all, so the pretalx docs don't include examples for it. Having this linked here will hopefully help others in the future!

**Reply by edgarallan** [reply]
Hi Carolrune, I'm using Apache too and I'm having problems (404 error from browser) in all .css an other files in /media
Can you please post you virtual host configuration entirely?
Thanks!
